src/components/DocumentationTopic/DocumentationNav/LanguageToggle.vue

```diff
@@ -23,7 +23,7 @@
       <label
         :for="hasLanguages ? 'language-toggle' : null"
         class="nav-menu-setting-label"
-      >Language:</label>
+      >{{ $t('language') }}</label>
```

Comment:
It doesn't seem like the actual localized string included the colon. Will that be update later? I think we should at least make sure it's added in English (since we will introduce a regression otherwise). For Mandarin and Japanese we need to make sure they are added there (if needed)

Reply:
It should, I'm waiting for the translations